Makefile

```diff
@@ -1,3 +1,4 @@
+SHELL=/usr/bin/bash
```

Comment:
Does something go wrong if you remove this?  My machine doesn't have bash at this location.  I tried to fix it by doing `SHELL=$(shell which -a bash)` but it complains `Makefile:1: *** Recursive variable 'SHELL' references itself (eventually).  Stop.`.

Reply:
It also works for me when I write bash